docs/sphinx_api_docs_source/utils.py

```diff
@@ -35,3 +35,30 @@ def apply_markdown_adjustments(soup, html_file_path, html_file_contents):  # noq
         if lengthChildren > 0:
             item.insert(0, "\r\n")
             item.append("\r\n")
+
+
+def apply_structure_changes(soup, html_file_path, html_file_contents):
+    # Add h2 title to Methods section
+    methods = soup.select(".py.method")
+    if methods:
+        add_section_title(soup, "Methods", methods)
```

Comment:
Nit. Can we switch the order of the params? It's a more common practice IMO.
```suggestion
        add_section_title(soup, methods, "Methods")
```

Reply:
Sure!